### ISSUE TITLE
google_metadata_script_runner fails with u'\u2019' characters

### ISSUE THREAD

**Issue by uberspot** [GoogleCloudPlatform/compute-image-packages#801]
As taken from `/var/log/messages` from 
```
Jul  5 15:49:24 testinstance google_metadata_script_runner: File "/usr/bin/google_metadata_script_runner", line 9, in <module>
Jul  5 15:49:24 testinstance google_metadata_script_runner: load_entry_point('google-compute-engine==2.8.16', 'console_scripts', 'google_metadata_script_runner')()
Jul  5 15:49:24 testinstance google_metadata_script_runner: File "/usr/lib/python2.7/site-packages/google_compute_engine/metadata_scripts/script_manager.py", line 109, in main
Jul  5 15:49:24 testinstance google_metadata_script_runner: debug=bool(options.debug))
Jul  5 15:49:24 testinstance google_metadata_script_runner: File "/usr/lib/python2.7/site-packages/google_compute_engine/metadata_scripts/script_manager.py", line 69, in __init__
Jul  5 15:49:24 testinstance google_metadata_script_runner: self._RunScripts(run_dir=run_dir)
Jul  5 15:49:24 testinstance google_metadata_script_runner: File "/usr/lib/python2.7/site-packages/google_compute_engine/metadata_scripts/script_manager.py", line 80, in _RunScripts
Jul  5 15:49:24 testinstance google_metadata_script_runner: script_dict = self.retriever.GetScripts(dest_dir)
Jul  5 15:49:24 testinstance google_metadata_script_runner: File "/usr/lib/python2.7/site-packages/google_compute_engine/metadata_scripts/script_retriever.py", line 256, in GetScripts
Jul  5 15:49:24 testinstance google-clock-skew: INFO Starting Google Clock Skew daemon.
Jul  5 15:49:24 testinstance google_metadata_script_runner: return (self._GetAttributeScripts(instance_data, dest_dir)
Jul  5 15:49:24 testinstance google_metadata_script_runner: File "/usr/lib/python2.7/site-packages/google_compute_engine/metadata_scripts/script_retriever.py", line 219, in _GetAttributeScripts
Jul  5 15:49:24 testinstance google_metadata_script_runner: dest.write(metadata_value.lstrip())
Jul  5 15:49:24 testinstance google_metadata_script_runner: UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 2478: ordinal not in range(128)

```

```
uname -a
Linux testinstance 3.10.0-957.21.3.el7.x86_64 #1 SMP Tue Jun 18 16:35:19 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

```

based on the `gce-uefi-images/centos-7` gcp image.

**Comment by illfelder** [comment]
Were you using gcloud? If so, we believe there was a regression that was fixed in the latest version of gcloud.

**Comment by uberspot** [comment]
I was using terraform with the latest version of google-provider.

**Comment by zmarano** [comment]
This is definitely not related to gcloud. We can reproduce this without gcloud involved. Looking into it more.

**Comment by zmarano** [comment]
Fixed in #822
